NodleBar/app/src/main/java/com/elysium/nodlebar/BasketActivity.java

```diff
@@ -0,0 +1,268 @@
+package com.elysium.nodlebar;
+
+import android.app.Dialog;
+import android.content.DialogInterface;
+import android.os.Bundle;
+import android.support.design.widget.Snackbar;
+import android.support.v4.app.DialogFragment;
+import android.support.v7.app.AlertDialog;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.support.v7.widget.Toolbar;
+import android.util.Log;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.Button;
+import android.widget.Toast;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Locale;
+
+public class BasketActivity extends AppCompatActivity implements View.OnClickListener, BasketListRecyclerViewAdapter.OnChangeQuantityListener{
+    RecyclerView mRecyclerView;
+    Button btnBack, btnCheckOut, btnRemoveAll;
+    BasketListRecyclerViewAdapter adapter;
+    UserBasket basket;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_basket);
+        setTitle("Basket");
+
+        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
+        //setSupportActionBar(toolbar); Threw an error...
```

Comment:
This happened because BasketActivity is using the default AppTheme, which includes an ActionBar. If you want to include your own Toolbar in the XML and set it as an ActionBar here, then in the manifest you have to set the theme for the activity to be AppTheme.NoActionBar